docs/EN/CompatiblePumps/OmnipodDASH.md

```diff
@@ -36,7 +36,12 @@ The instructions below explain how to activate a new pod session. Wait to close
 
 ## Before You Begin
 
-**SAFETY FIRST** - you should not try to connect **AAPS** to a pod for the first time without having access to extra pods, insulin, and phone devices are a must have.
+**SAFETY FIRST** - you should not try to connect **AAPS** to a pod for the first time without having access to all of the following:
+1. Extra pods (3 or more spare)
+2. Spare Insulin and MDI equipment
+3. A working Omnipod PDM (In case **AAPS** fails)
+4. Supported Phones are a must!
+5. Correct version of AAPS built and installed (See [Hardware/Software Requirements](#Hardware/Software-Requirements))
```

Comment:
That link won't work. I added a new anchor (omnipod-dash-hardware-software-requirements) to make sure #hardware-software-requirements won't be broken by translation